.travis.yml

```diff
@@ -6,11 +6,15 @@ jobs:
     # macOS
     - env: TARGET_MACHINE=i3osx
       os: osx
-    - env: TARGET_MACHINE=ti3osx
+    - env:
+      - TARGET_MACHINE=ti3osx
+      - PARALLEL_MATS=1
```

Comment:
I assume it’s the test machines, since I had no problems running on my intel hardware. The test machines have only two cores, and I don’t know how many concurrent threads the threads mat spawns... but I can believe a sufficiently underpowered VM could run in to trouble and time out. Or if there are any tests that have one thread busy-waiting then running two builds could starve.